matrix/lambdas/daemons/mapper.py

```diff
@@ -68,7 +69,7 @@ def _get_worker_payload(self, worker_chunk_spec: dict) -> dict:
         }
 
     @staticmethod
-    def _get_chunk_specs(bundle_uuid: str, bundle_version: str) -> typing.List[dict]:
+    def _get_chunk_specs(bundle_uuids: typing.List[str], bundle_versions: typing.List[str]) -> typing.List[dict]:
```

Comment:
Actually I did want to (and will) to refactor this to operate on a single bundle and run inside the for loop. Just noting

Reply:
:+1: